ros/src/tl_detector/tl_detector.py

```diff
@@ -19,11 +19,13 @@
 
 class TLDetector(object):
 
-    def __init__(self, init_without_ros=False):
+    def __init__(self, is_testing=False):
         """
-        :param init_without_ros: in case class has to run in test environment and we don't want ros sub/pub to fail
+        :param is_testing: in case class has to run in test environment without ros, and we don't want ros sub/pub to fail
         """
 
+        self.is_testing = is_testing
```

Comment:
I agree, which is why I was not comfortable with the changes I was making last night.

I've kept this name, since I think it's clearer than the "ROS not initialized" name. Now it only protects two blocks of code, the block in the constructor and the "return light.state" hack in get_light_state(), which can't be avoided for unit testing.